### PR TITLE
#17954 update refs on copy template

### PR DESCRIPTION
**Pull request by fabrizzio-dotCMS**
Both
Template drownBody ( Regular templates )
Template body ( Advanced Templates )
needed to update their references to point to the new containers copied during a site copy.
drowned Body was previously handled but for advanced templates, we needed to work through the filed body. Replacing all occurrences of things like 
`#parseContainer('d71d56b4-0a8b-4bb2-be15-ffa5a23366ea','1234578998555')`
and make the uuid substitution to point to the newly copied template.

### CODE REVIEW

File: dotCMS/src/main/java/com/dotmarketing/util/StringUtils.java [hunk 1]
```diff
@@ -316,7 +318,25 @@ public static String sanitizeFileName(final String fileName) {
       return FileUtil.sanitizeFileName(fileName);
       
     }
-    
+
+    private static final Pattern singleQuotedLiteral = Pattern.compile("(?:^|\\s)*'([a-zA-Z0-9-_]+)'(?:$|\\s)*", Pattern.MULTILINE);
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotmarketing/util/StringUtils.java]: Codacy Issue found: [Field singleQuotedLiteral has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5002120)

File: dotCMS/src/test/java/com/dotmarketing/util/StringUtilsTest.java [hunk 2]
```diff
@@ -315,4 +318,26 @@ public void testLowercaseStringExceptMatchingTokens(final String query,
...
+                {"Tumblr's la la la '123 456' text can be hard to parse. ", 0}, //<-- by design literals with blanks are excluded.
+                { "## Container: Blank Container\n"
+                + "## This is autogenerated code that cannot be changed\n"
+                + "#parseContainer('d71d56b4-0a8b-4bb2-be15-ffa5a23366ea','1539784124854')\n", 2}
+        };
+    }
+
+    @Test
+    @UseDataProvider("singleQuotedText")
+    public void testSingleQuotedText(String textBlock, int expectedResult){
```
Comment (dev-dotcms) [inline dotCMS/src/test/java/com/dotmarketing/util/StringUtilsTest.java]: Codacy Issue found: [Parameter 'textBlock' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5002120)

**Review by jdotcms — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]
